scripts/release_scripts/get_repo_specific_changes.py

```diff
@@ -0,0 +1,151 @@
+
+    Returns:
+        list(str). List of variable names that changed.
+    """
+    feconf_changed_version = []
+    git_show_cmd = (GIT_CMD_SHOW_FORMAT_STRING % current_release)
+    old_feconf = common.run_cmd(git_show_cmd.split(' '))
+    with python_utils.open_file(FECONF_FILEPATH, 'r') as feconf_file:
+        new_feconf = feconf_file.read()
+    for variable in FECONF_VAR_NAMES:
+        old_version = re.findall(
+            VERSION_RE_FORMAT_STRING % variable, old_feconf)[0]
+        new_version = re.findall(
+            VERSION_RE_FORMAT_STRING % variable, new_feconf)[0]
+        if old_version != new_version:
+            feconf_changed_version.append(variable)
+    return feconf_changed_version
+
+
+def _git_diff_names_only(left, right='HEAD'):
+    """Get names of changed files from git.
+
+    Args:
+        left: str. Lefthand timepoint.
+        right: str. Rightand timepoint.
+
+    Returns:
+        list(str). List of files that are different between the two points.
+    """
+    diff_cmd = (GIT_CMD_DIFF_NAMES_ONLY_FORMAT_STRING % (left, right))
+    return common.run_cmd(diff_cmd.split(' ')).splitlines()
+
+
+def get_changes_in_setup_scripts(base_release_tag, changed_only=True):
+    """Check if setup scripts have changed.
+
+    Args:
+        base_release_tag: str. The current release tag to diff against.
+        changed_only: bool. If set to False will return all tested files
+            instead of just the changed ones.
+
+    Returns:
+        dict. Dict consisting of key as script name and value as boolean
+        indicating whether or not it has changed
+        (filtered by default to those that are modified).
+    """
+    setup_scripts = ['scripts/%s' % item for item in
+                     ['setup.py', 'setup_gae.py', 'install_third_party_libs.py',
+                      'install_third_party.py']]
+    changed_files = _git_diff_names_only(base_release_tag)
+    changes_dict = {script: script in changed_files
+                    for script in setup_scripts}
+    if changed_only:
+        return {name: status for name, status
+                in changes_dict.items() if status}
+    else:
+        return changes_dict
+
+
+def get_changes_in_storage_models(current_release):
+    """Returns a list of filepaths in core/storage whose contents have
+    changed since the last release.
+
+    Args:
+        current_release: str. The current release version.
+
+    Returns:
+        list(str). The changed files in core/storage (if any).
+    """
+    diff_list = _git_diff_names_only(current_release)
+    return [item for item in diff_list if item.startswith('core/storage')]
+
+
+def get_changes(current_release):
+    """Collects changes in storage models, setup scripts and feconf
+    since the previous release.
+
+    Args:
+        current_release: str. The current release version.
+
+    Returns:
+        list(str). The list of changed storage models, setup scripts and
```

Comment:
This seems like a polymorphic list, but it's just a list of undifferentiated strings, right?

If so, make the docstring clearer, e.g.: A list of lines to be written to the release summary file. These lines describe XXX, YYY and ZZZ.

Reply:
Done.